app/routes/register-your-interest/index.js

```diff
@@ -58,8 +59,23 @@ module.exports = [
       }
     },
     handler: async (request, h) => {
-      await sendEmail(registerYourInterest, request.payload.emailAddress)
-      await sendDefraIdRegisterYourInterestMessage(request.payload.emailAddress)
+      const { alreadyRegistered, accessGranted } = await checkWaitingList(request.payload.emailAddress)
+      const emailTemplate = alreadyRegistered && accessGranted ? config.notifyConfig.emailTemplates.accessGranted : config.notifyConfig.emailTemplates.accessNotGranted
+      if (!alreadyRegistered) {
+        sendEmail(config.notifyConfig.emailTemplates.registerYourInterest, request.payload.emailAddress)
+        sendDefraIdRegisterYourInterestMessage(request.payload.emailAddress)
+      } else {
+        sendEmail(
+          emailTemplate,
```

Comment:
it can possibly be written like this: sendEmail(accessGranted ? config.notifyConfig.emailTemplates.accessGranted : config.notifyConfig.emailTemplates.accessNotGranted, ....